source/material_model/visco_plastic.cc

```diff
@@ -297,6 +297,27 @@ namespace aspect
                   elastic_out->elastic_shear_moduli[i] = average_elastic_shear_moduli[i];
                 }
             }
+
+          // Prescribed dilation
+          double average_friction_angle = 0.;
+          double dilation_rate = 0.;
+          if (this->get_parameters().enable_prescribed_dilation)
+            {
+              MaterialModel::PrescribedPlasticDilation<dim>
+              *prescribed_dilation = out.template get_additional_output<MaterialModel::PrescribedPlasticDilation<dim>>();
+
+              if (prescribed_dilation)
```

Comment:
should this be an error if you enable dilation but you don't have the outputs? Or does this break somewhere?

Reply:
My thought was we would simply not fill the outputs if they don't exist with this logic. Is there a better way to ensure this is the case?